lib/PlaidClient.js

```diff
@@ -90,11 +90,14 @@ Client.prototype.createItemAddToken =
       options = {};
     }
     const {
+      client_user_id,
       user_identity,
     } = options || {};
     return this._authenticatedRequest({
       path: '/item/add_token/create',
-      body: helpers.processUserFields(user_identity),
+      body: R.merge(
```

Comment:
What do you think about merging `client_user_id` into `user_identity` since we do something similar in the API?

Reply:
⚡ 689caa5